source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -348,7 +348,7 @@ def _get_windowThreadID(self):
 			threadID = super().windowThreadID
 		return threadID
 
-	def _get_is21H1Plus(self):
+	def _get_isPostUIARefactor(self):
```

Comment:
This isn't very descriptive. If the main feature that consoleUIATextInfo relies on is GetVisibleRanges then perhaps it should be called 'isGetVisibleRangeAvailable`

Reply:
This, too, is somewhat misleading I think. `GetVisibleRanges` was available in the bad UIA implementation as well, but it returned incorrect results in some cases.